### PR TITLE
Project 2 - Responsive Design - Milestone Two - Finish

### PR DESCRIPTION
**Pull request by jmontanholi**
# The implementations done in this Branch were:

- Categories HTML and CSS
- Latest News HTML and CSS
- Latest Deals HTML and CSS
- All three with Responsive Queries (for Mobile, Ipad, and Desktop)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,9 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (christianotieno) [inline README.md:1]: ## Improve the live link of the page on the `README` 
- [x] Please add a live link to where the page is hosted live. The current link appears to be broken as it throws a `404` error upon visiting the link.

File: index.html [hunk 2]
```diff
@@ -115,9 +114,123 @@
         </div>
       </div>
     </div> 
-    
-    <!-- LATEST FUNDING -->
 
+    <!-- Latest News -->
```
Comment (christianotieno) [inline index.html:118]: ## Improve the `general layout` of the page
- [x] The `general layout` of the page from top to bottom are not aligned as viewed or compared with the original page. You should note that the first main three articles must not cover the same width as that of the rest of the contents of the page. The width is much wider in this section as the rest of the section gets to be uniform, lesser and centred. Have a look at the comparison below for inspiration. 👇 
- [x] Be mindful of the spacing around elements and the font sizes as well. They appear much bolder on the source page.

|[Source Page](https://perma.cc/M5ZV-Q2D6)|[Clone Page](https://raw.githack.com/jmontanholi/ProjectTwoMilestoneTwo/categories/index.html)|
|:--:|:--:|
|tnw|tnw-clone|

File: queries.css [hunk 3]
```diff
@@ -119,6 +118,51 @@
...
+
+  .ld-content {
+    flex-direction: column;
+    margin-right: 10px;
+  }
+
+  .ld-img img {
+    height: 120px;
+    width: 100%;
+  }
```
Comment (christianotieno) [inline queries.css]: ## Aim to Stay `DRY`
- [x] The lines of code on line `#146` & `#162` appear to be similar and make you seem like you are repeating yourself. Would you mind merging them so as to not appear redundant?

File: queries.css [hunk 4]
```diff
@@ -168,4 +212,37 @@
...
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: repeat(3, 1fr);
+  }
+
+  .ln-grid {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+  }
```
Comment (christianotieno) [inline queries.css:227]: ## Thumbs up 👍 
✅ Great job showcasing the greatness that comes with arranging items on the UI using the `grid` box rule. I love how you pulled this off in a neat and minimalistic way. Awesome stuff.

File: styles.css [hunk 5]
```diff
@@ -405,8 +450,220 @@ a.active {
...
+}
+
+.ld-content {
+  display: flex;
+  margin-top: 10px;
+}
+
+.ld-txt {
+  margin-left: 10px;
+}
```
Comment (christianotieno) [inline styles.css]: ## Aim to Stay `DRY`
- [x] The lines of code on line `#628` & `#648` appear to be similar and make you seem like you are repeating yourself. Would you mind merging them so as to not appear redundant?

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 
Hello @jmontanholi, @errea & @mrigorir 👋 


The codebase is greatly inspiring with how you have implemented the layout using CSS and HTML. Most of the requirements are getting met and the hard work in putting out a professionally kept repo is evident from the commit messages to the Pull Request. Keep it up.

Below, 👇 I have highlighted some few suggestions that might require your attention.

The changes touch on having to provide a live link to where the page is hosted live to getting the general layout right to finally some suggestions on some code that appear to be redundant. Take a look and tell me what you think. 

### Happy coding 💻 
#### Christianb Otieno (CR)

**Review by Georjane — APPROVED** [review]
## Status: PR Approved ✔️ ✔️
### Hi Team 🙌
Very nice work you have there. Congratulations on successfully completing this project. 🎊 It meets the minimum requirements for approval. 

It would be very nice to have a screenshot of your project in your Readme file too. Other than that, you can go ahead and merge this PR. 👍 

Good luck on your next project and Happy Coding 💻

**Comment by errea** [comment]
👍Well-done @jmontanholi 🙌 we did a great job.

**Pull request merged**